pkg/apis/hco/v1alpha1/hyperconverged_types.go

```diff
@@ -13,6 +13,8 @@ type HyperConvergedSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+    KubeVirtImagePullPolicy	string	`json:"KubeVirtImagePullPolicy"`
```

Comment:
I think you'll need to add `omitempty` to each of these vars to prevent `KubeVirtImagePullPolicy: ""`, which is currently a valid input.

Reply:
Fixed, even though I'm checking in the code for empty string, and thus empty string has basically the same effect as not including the key.